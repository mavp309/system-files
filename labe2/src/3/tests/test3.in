7 2147483647 2147483647 2 42 4 42 3 42 2147483647 2147483647 2147483647 2147483647 2147483647 2147483647
