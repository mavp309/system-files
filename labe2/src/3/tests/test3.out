281729 511104 298816 359209 281729 281729 281729 
